shell/src/main/java/org/apache/accumulo/shell/commands/CreateNamespaceCommand.java

```diff
@@ -37,33 +37,42 @@
 
 public class CreateNamespaceCommand extends Command {
   private Option createNamespaceOptCopyConfig;
+  private Option createNamespaceOptCopyProperties;
 
   @Override
   public int execute(final String fullCommand, final CommandLine cl, final Shell shellState)
       throws AccumuloException, AccumuloSecurityException, TableExistsException,
       TableNotFoundException, IOException, ClassNotFoundException, NamespaceExistsException,
       NamespaceNotFoundException {
 
-    if (createNamespaceOptCopyConfig == null) {
-      getOptions();
-    }
+    getOptions();
 
     String namespace = cl.getArgs()[0];
 
     shellState.getAccumuloClient().namespaceOperations().create(namespace);
 
-    // Copy options if flag was set
-    Map<String,String> configuration = null;
+    Map<String,String> propsToSet = null;
+
+    // Copy configuration options if flag was set
     if (cl.hasOption(createNamespaceOptCopyConfig.getOpt())) {
-      String copy = cl.getOptionValue(createNamespaceOptCopyConfig.getOpt());
+      String srcNs = cl.getOptionValue(createNamespaceOptCopyConfig.getOpt());
       if (shellState.getAccumuloClient().namespaceOperations().exists(namespace)) {
-        configuration = shellState.getAccumuloClient().namespaceOperations().getConfiguration(copy);
+        propsToSet = shellState.getAccumuloClient().namespaceOperations().getConfiguration(srcNs);
       }
     }
-    if (configuration != null) {
-      final Map<String,String> config = configuration;
+
+    if (cl.hasOption(createNamespaceOptCopyProperties.getOpt())) {
+      String srcNs = cl.getOptionValue(createNamespaceOptCopyProperties.getOpt());
+      if (shellState.getAccumuloClient().namespaceOperations().exists(namespace)) {
+        propsToSet =
+            shellState.getAccumuloClient().namespaceOperations().getNamespaceProperties(srcNs);
+      }
+    }
```

Comment:
I didn't realize we already had an API that got only the namespace properties. I thought the new API showed the effective properties, just like the old API.